Automattic/jetpack
# Embeds: Add support for Instagram Reel permalinks

Fixes #17662 

#### Changes proposed in this Pull Request:

Adds support for Instagram Reel permalinks to be used as embedded content.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:






1. Create a new post and add an Instagram block
2. Enter the URL https://instagram.com/reel/COWmlFLB_7P/
3. See that the Instagram Reel embed is now in the post

Before | After
---|---
![image](https://user-images.githubusercontent.com/16200219/116935955-a7f5c980-ac56-11eb-834e-2cf95406b059.png) | ![image](https://user-images.githubusercontent.com/16200219/116936131-e4292a00-ac56-11eb-9974-93b32d71a26f.png)
